Rsk.Samples.IdentityServer4.AdminUiIntegration/Program.cs

```diff
@@ -18,11 +26,27 @@ public static void Main(string[] args)
         {
             var host = BuildWebHost(args).Build();
 
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", true, true)
+                .AddEnvironmentVariables();
+            var configuration = builder.Build();
+
             if (args.Contains("seed"))
             {
-                var context = host.Services.GetRequiredService<IConfigurationDbContext>();
-                var exitCode = Config.Seed(context);
-                Environment.ExitCode = exitCode;
```

Comment:
Do we want to still use exit codes?

Reply:
I'm not using them for flow in the container startup so i'll remove it.